### PR TITLE
Created a Pull Request Template

### PR DESCRIPTION
**Pull request by kylemasc917**
Created a pull request template that can be used for this assignment and others going forward.

### CODE REVIEW

File: PullRequestTemplate.md
Comment (laserlab) [inline PullRequestTemplate.md:1]: I would add headers for placeholders so people can put in missing content like: 

```markdown
# Pull Request Template

For this assignment all pull requests going  forward should contain the following.

## Title
  -A ledgible title that briefly describes the contents of the the pull request.
  -A ledgible description tht indepthly describes the contents of the pull request.

### Contents
  -The contents that you have been assigned to submit labeled with titles that acuratly describe the assignment they are for. i.e. the csv file of the 9 groups should be labeled: NineGroups.csv or something similar.

### Labels Milestones
  -ALL pull requests must have an attatched label and Milestone before a reviewer can be assigned.

Any further questions on formatting pull requests can be asked in the issues tab or on discord. 
This document can be changed at any time.
```

**Review by laserlab — COMMENTED** [review]
add headers. Can also be done in separate PR to speed up merging.

**Comment by kylemasc917** [comment]
Ok edits have been made

**Comment by HeshamElsaman** [comment]
Great, looks good.
Merging.

**Review by HeshamElsaman — COMMENTED** [review]
I forgot to submit my review, apologies for that.

**Pull request merged**